indexer/db/solana_block_db.py

```diff
@@ -312,6 +335,12 @@ async def activate_block_list(self, ctx: DbTxCtx, from_slot: int, slot_list: tup
         await self._update_row(ctx, self._deactivate_query, by_slot_range)
         await self._update_row(ctx, self._activate_query, by_slot_range)
 
+    async def get_priority_fee_percentiles(self, ctx: DbTxCtx, num_blocks: int) -> list[list[int]]:
```

Comment:
Please rename to:
`get_cu_price_percentile_list`

Reply:
done